include/SSVOpenHexagon/Core/HexagonDialogBox.hpp

```diff
@@ -17,31 +17,67 @@ namespace hg
 class HGAssets;
 class StyleData;
 
+enum DBoxDraw
+{
+    topLeft = 0,
+    center,
+    centerUpperHalf
+};
+
 class HexagonDialogBox
 {
 private:
+    using KKey = sf::Keyboard::Key;
+    using Color = sf::Color;
+    using Vector2 = sf::Vector2f;
+
     HGAssets& assets;
     ssvs::GameWindow& window;
     StyleData& styleData;
     sf::Font& imagine;
 
+    std::function<void(const Color&, const Color&)> drawFunc;
```

Comment:
This is one of the places where I would use an alias, as I suggest some refactoring below.

```suggestion
    using DrawFunc = std::function<void(const Color&, const Color&)>;
    DrawFunc drawFunc;
```